test/io/sungeun/ioerror.chpl

```diff
@@ -1,5 +1,5 @@
 // Test IO error function interfaces
-use IO;
+use IO, OS.POSIX;
```

Comment:
This change also makes me wonder about the things that IO does/should re-export...

Reply:
It's interesting, because the IO module does not provide `ioerror`, that's defined in SysError.